assets/dev/js/frontend/tools/stretch-element.js

```diff
@@ -16,11 +17,11 @@ module.exports = elementorModules.ViewModule.extend( {
 	},
 
 	stretch() {
-		var containerSelector = this.getSettings( 'selectors.container' ),
+		var settings = this.getSettings(),
```

Comment:
Should we not be using `let` or `const`
or are we not changing this in older files?

Reply:
Done.